src/main/java/org/prebid/server/bidder/undertone/proto/UndertoneRequestExt.java

```diff
@@ -0,0 +1,17 @@
+package org.prebid.server.bidder.undertone.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class UndertoneRequestExt {
+
+    @JsonProperty("id")
+    int adapterId;
+
+    @JsonProperty("version")
```

Comment:
No need of `@JsonProperty("version")` because variable name is similar, so it will serialize to `version` even without annotation